Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/SwitchTeamMessage.java

```diff
@@ -1,42 +1,41 @@
-package nl.soccar.gameserver.message;
-
-import nl.soccar.gameserver.message.handler.SwitchTeamMessageHandler;
-import nl.soccar.library.enumeration.TeamColour;
-import nl.soccar.socnet.message.Message;
-import nl.soccar.socnet.message.MessageConstants;
-import nl.soccar.socnet.message.MessageEvent;
-
-/**
- *
- * @author PTS34A
- */
-@MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
-public final class SwitchTeamMessage extends Message {
-
-    private final int playerId;
-    private final TeamColour team;
-
-    public SwitchTeamMessage(int playerId, TeamColour team) {
-        this.playerId = playerId;
-        this.team = team;
-    }
-    
-    public SwitchTeamMessage() {
-        playerId = -1;
-        team = null;
-    }
-
-    public int getPlayerId() {
-        return playerId;
-    }
-    
-    public TeamColour getTeamColour() {
-        return team;
-    }
-    
-    @Override
-    public int getId() {
-        return MessageConstants.SWITCH_TEAM_MESSAGE_ID;
-    }
-
-}
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.SwitchTeamMessageHandler;
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
+public final class SwitchTeamMessage extends Message {
+
+    private final int playerId;
+    private final TeamColour team;
+
+    public SwitchTeamMessage(int playerId, TeamColour team) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)